src/ReflectionMethod.php

```diff
@@ -114,7 +115,7 @@ public function __toString()
             $this->getEndLine(),
             count($methodParameters),
             $paramString,
-            $hasReturnType ? ReflectionType::convertToDisplayType($this->getReturnType()) : ''
+            $returnType ? ReflectionType::convertToDisplayType($returnType) : ''
```

Comment:
Ok... I confirmed that both cases are tested... (I did not flag the specific tests, but they're easy to find.) When `$returnType` is `null` the empty string value isn't used, but it prevents the `convertToDisplayType()` static method call that would be illegal with a `null` parameter. I am replacing the empty string with a string explaining its purpose.